lib/cardano_explorer/genesis.rb

```diff
@@ -0,0 +1,32 @@
+module CardanoExplorer
+  class Genesis < Base
+
+    def self.find
+      raise NoMethodError
+    end
+
+    def initialize
+    end
```

Comment:
What do you think if in the `Base` class, we make the argument in `initialize` optional?
```ruby
# base.rb
def initialize(hash = nil)
  @hash = hash
end
```

Then we do not need to redefine initialize here.

Reply:
Yep, sounds good. Updated